services/src/main/java/org/keycloak/utils/SecureContextResolver.java

```diff
@@ -1,4 +1,7 @@
-package org.keycloak.common.util;
```

Comment:
Just want to double check on how public are keycloak common apis? Do we have to give any consideration to backwards compatibility / deprecation in general?

Reply:
This file was moved here by mistake previously, and should not be part of common.

Reply:
Also considering this work will be in the upcoming patch release I don't think it will be too much of an issue.